tests/unittests/OperatorTest.cpp

```diff
@@ -1025,6 +1025,42 @@ TEST_P(OperatorTest, ReluSimple_Float16) {
   testReluSimple<float16_t>(bindings_, mod_, F_, EE_, ElemKind::Float16Ty);
 }
 
+/// Helper to test ReluSimple using \p DTy.
+template <typename DataType>
+static void testPReluSimple(glow::PlaceholderBindings &bindings,
+                            glow::Module &mod, glow::Function *F,
+                            glow::ExecutionEngine &EE, ElemKind DTy) {
+  auto *in = mod.createPlaceholder(DTy, {7}, "in", false);
+  auto *slope = mod.createPlaceholder(DTy, {7}, "slope", false);
+  auto *prelu = F->createPRELU("prelu", in, slope);
+  auto *save = F->createSave("prelu", prelu);
+  auto *result = bindings.allocate(save->getPlaceholder());
+
+  bindings.allocate(in)->getHandle<DataType>() = {0, -1, -2, -3, 4, 5, 6};
+  bindings.allocate(slope)->getHandle<DataType>() = {1, 1, 1, 1, 1, 1, 1};
+
+  EE.compile(CompilationMode::Infer, F);
+  EE.run(bindings);
+
+  auto resultH = result->getHandle<DataType>();
+  auto inH = bindings.get(in)->getHandle<DataType>();
+
+  for (size_t i = 0; i < 7; i++) {
+    EXPECT_EQ(resultH.raw(i), inH.raw(i));
+  }
+}
+
+/// Verify that the PRELU operator works correctly for Float.
+TEST_P(OperatorTest, PReluSimple_Float) {
+  testPReluSimple<float>(bindings_, mod_, F_, EE_, ElemKind::FloatTy);
+}
+
+/// Verify that the PRELU operator works correctly for Float16.
+TEST_P(OperatorTest, PReluSimple_Float16) {
```

Comment:
I see ReLU follows this, but Name_Something is not the convention we follow (remove _)

Reply:
I actually added underscores to some unit tests to specify types. We can remove them if desired -- but I thought it was a bit cleaner to separate operator from type being tested.

Reply:
that's ok.